lib/themes/dosomething/paraneue_dosomething/templates/system/patterns/photo.tpl.php

```diff
@@ -1,4 +1,9 @@
 <figure class="photo -stacked -framed">
+  <?php if (user_access('view any reportback')): ?>
+    <div class="admin-edit">
+      <a class="button -secondary" href="/admin/reportback/<?php print $content['rbid'] ?>">Edit Status</a>
```

Comment:
can we pass this in from the module? both the user_access and the link?
then in the template we would just say 
```
<?php if (!is_null($admin_link): ?>
 print $admin_link;
<?php endif; ?>
```